src/common/ini_handler.c

```diff
@@ -53,9 +81,9 @@ uint8_t ini_save_file(const char *ini_save_str) {
     UINT written_bytes = 0;
     FIL ini_file;
 
-    f_unlink(network_ini_file_name);
+    f_unlink(ini_file_name);
```

Comment:
when you are already using the POSIX FILE* for the INI parsing, please update this old FatFs code into the POSIX file interface

Reply:
fixed